grc/scripts/freedesktop/org.gnuradio.grc.metainfo.xml

```diff
@@ -50,11 +50,11 @@
 <url type="homepage">https://www.gnuradio.org</url>
 <url type="help">https://wiki.gnuradio.org/</url>
 <url type="bugtracker">https://github.com/gnuradio/gnuradio/issues</url>
-<url type="contact">mailto:discuss-gnuradio@gnu.org</url>
 <url type="contact">https://lists.gnu.org/mailman/listinfo/discuss-gnuradio</url>
 <url type="contact">https://chat.gnuradio.org</url>
 <provides>
   <binary>gnuradio-companion</binary>
   <binary>grcc</binary>
 </provides>
+<content_rating type="oars-1.1" />
```

Comment:
I'd agree, we're absolutely suited for all audiences. However, this feels *super* unnecessary for a desktop application. Also, it places the burden on the maintainers to check for any feature we might add in the future (e.g. chat integration, flow graph sharing) whether we might need adjust this setting – again, for no obvious (to me) gain.

I'd rather we not do that. Not a strongly held opinion, though

Reply:
Well, the OARS metadata that I added are empty (which usually means "suited for all audiences" vs "unknown" when there are no OARS metadata). :-) As I mentioned, this is just to make the AppStream parsers happy because many of them now require OARS metadata and fail to parse the AppStream file when there are no OARS metadata in it. Many other desktop applications have already added empty OARS metadata to their AppStream files for the same reason.

Reply:
oh, many? I was aware of only one standard, and that does not mandate these tags. Which parsers fail? I'd rather fix their bugs upstream (namely, not accepting valid data) than adding a persistent maintenance burden downstream. Enforcing meaningless metadata that is likely to outdate is strictly a bug for a metadata parsing ecosystem!

Reply:
Don't take this the wrong way, please. The problem I'm having here is that there's a somewhat clear file format standard – adhering to anything stricter than that seems OK. But if tools consuming the kind of file refuse to work with files that adhere to the standard on own, undocumented rules, then that's a problem of these tools and should not be exported to projects contributing metadata :)

Reply:
There is only one standard specification, however OARS metadata are a special case. :-) They are [considered optional](https://freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-content_rating) by the standard, but since there was a lot of discussion and plans to make OARS mandatory in the future, some parsers and validators started requiring OARS. A good example of this is [Flathub](https://github.com/flathub/flathub/issues/158).

Reply:
Anyway, as I said, if you want, I can remove the OARS tag. I would like to at least keep the second part of the commit that removes the mailto url since that one was considered an error by appstreamcli validator and was actually never properly supported, but it is also up to you.

Reply:
By the way, for example [Gqrx](https://gqrx.dk) has the same OARS metadata: https://github.com/gqrx-sdr/gqrx/blob/master/dk.gqrx.gqrx.appdata.xml#L31